apps/heft/src/plugins/ApiExtractorPlugin/ApiExtractorRunner.ts

```diff
@@ -57,6 +58,17 @@ export class ApiExtractorRunner extends SubprocessRunnerBase<IApiExtractorRunner
 
     const apiExtractor: typeof TApiExtractor = require(this._configuration.apiExtractorPackagePath);
 
+    this._terminal.writeLine(`Using API Extractor version ${apiExtractor.Extractor.version}`);
```

Comment:
This is the global terminal. Do this instead:

```suggestion
    this._scopedLogger.terminal.writeLine(`Using API Extractor version ${apiExtractor.Extractor.version}`);
```